appstudio-controller/controllers/appstudio.redhat.com/snapshotenvironmentbinding_controller.go

```diff
@@ -244,6 +261,10 @@ func (r *SnapshotEnvironmentBindingReconciler) Reconcile(ctx context.Context, re
 		}
 	}
 
+	if allErrors != nil {
```

Comment:
I moved this from lower in the function, to higher in the function. It didn't seem to make much sense to continue if one of the GitOpsDeployments returned an error.